ocs_ci/ocs/constants.py

```diff
@@ -74,6 +74,10 @@
 # NooBaa statuses
 BS_AUTH_FAILED = 'AUTH_FAILED'
 BS_OPTIMAL = 'OPTIMAL'
+HEALTHY_OB = 'OPTIMAL'
+HEALTHY_OBC = STATUS_BOUND
+HEALTHY_OBC_CLI_PHASE = 'Phase                  : Bound'
```

Comment:
Is there really so many whitespaces between `Phase` and `:`? Isn't it a bug? Maybe it should be:
```
`'Phase:                   Bound'`
```

Reply:
Not a bug - it's used to prettify the CLI prints - 
```
Bucket status:
  Name                   : oc-bucket-1b267097d1ed4f40aab0eb041b5fdde1
  Type                   : REGULAR
  Mode                   : OPTIMAL
```

```
ObjectBucketClaim info:
  Phase                  : Bound
```

Reply:
I tried to come up with a way to parse the CLI output, but it's simply not machine readable enough. Thus, I chose to go with space stripping.

Reply:
Ok